src/libmongoc/src/mongoc/mongoc-server-api.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2021 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "mongoc-config.h"
+#include "mongoc-server-api-private.h"
+
+const char *
+mongoc_server_api_version_to_string (mongoc_server_api_version_t version)
+{
+   const char *out;
+
+   /* Until version numbers top 1000, this should be fine. After that point,
+      this method will need to be altered to hold memory for larger strings. */
+   bson_uint32_to_string (version, &out, NULL, 0);
```

Comment:
This assumes that API versions will always be numeric. I think it would make more sense to use a `switch` here to ensure we're not implicitly restricting future API versions.

Reply:
Done.